ql/src/test/results/clientpositive/llap/subquery_in.q.out

```diff
@@ -1805,22 +1833,38 @@ STAGE PLANS:
                   Filter Operator
                     predicate: _col1 is not null (type: boolean)
                     Statistics: Num rows: 13 Data size: 52 Basic stats: COMPLETE Column stats: COMPLETE
-                    Select Operator
-                      expressions: _col1 (type: int)
+                    Group By Operator
+                      keys: _col1 (type: int)
+                      minReductionHashAggr: 0.53846157
+                      mode: hash
                       outputColumnNames: _col0
-                      Statistics: Num rows: 13 Data size: 52 Basic stats: COMPLETE Column stats: COMPLETE
-                      Group By Operator
-                        keys: _col0 (type: int)
-                        minReductionHashAggr: 0.53846157
-                        mode: hash
-                        outputColumnNames: _col0
+                      Statistics: Num rows: 6 Data size: 24 Basic stats: COMPLETE Column stats: COMPLETE
+                      Reduce Output Operator
+                        key expressions: _col0 (type: int)
+                        null sort order: z
+                        sort order: +
+                        Map-reduce partition columns: _col0 (type: int)
                         Statistics: Num rows: 6 Data size: 24 Basic stats: COMPLETE Column stats: COMPLETE
-                        Reduce Output Operator
-                          key expressions: _col0 (type: int)
-                          null sort order: z
-                          sort order: +
-                          Map-reduce partition columns: _col0 (type: int)
-                          Statistics: Num rows: 6 Data size: 24 Basic stats: COMPLETE Column stats: COMPLETE
+        Reducer 4 
+            Execution mode: vectorized, llap
```

Comment:
Same as above: GBY is not being removed

Reply:
This one is slightly different, it needs further exploration. We end up with an additional group by (map+reduce side). This may be due to change in Calcite plan.